kitchen.macos.yml

```diff
@@ -0,0 +1,36 @@
+---
+driver:
+  name: exec
+
+provisioner:
+  sudo: true
+  salt_bootstrap_options: -MP stable %s
+  salt_call_command: /opt/salt/bin/salt-call
+  init_environment: |
+    echo 'auto_accept: true' > /tmp/auto-accept-keys.conf
+    sudo mkdir -p /etc/salt/master.d
+    sudo mv /tmp/auto-accept-keys.conf /etc/salt/master.d/auto-accept-keys.conf
+    brew install coreutils
+    sh -c 't=$(gshuf -i 1-15 -n 1); echo Sleeping $t seconds; sleep $t'
+
+platforms:
+  - name: macos-12
+  - name: macos-11
+  - name: macos-1015
+
+suites:
+  - name: py3-stable-3002
+    provisioner:
+      salt_version: 3002.8
+  - name: py3-stable-3003
+    provisioner:
+      salt_version: 3003.4
+  - name: py3-stable-3004
+    provisioner:
+      salt_version: 3004.1
+  - name: latest
+    provisioner:
+      salt_version: latest
+
+verifier:
+  command: pytest --cache-clear -v -s -ra --log-cli-level=debug -k "not test_ping" tests/integration/
```

Comment:
The `"not test_ping"` is for essentially the same reason as https://github.com/saltstack/salt-bootstrap/pull/1548#discussion_r602857775.